lib/datasource/remote/api/esr_callback_repository.dart

```diff
@@ -34,11 +34,11 @@ import 'package:seeds/datasource/remote/api/http_repo/http_repository.dart';
 /// At the moment can't justify the effort for a generic implementation.
 
 class ESRCallbackRepository extends HttpRepository {
-  Future<Result<bool>> callback(String _callbackUrl, String transactionId) {
-    print("[http] issue callback $_callbackUrl");
+  Future<Result<bool>> callback(String callbackUrl, String transactionId) {
```

Comment:
the original code here was slightly cleaner... if _ is no longer allowed, I would suggest removing the whole callbackURL (upper case URL) variable to avoid confusion, or rename it to "parsedUrl" 

Having a variable name only different by upper/lower case is not good.

Reply:
The variable name change was necessary because the parameter `_callbackUrl` and the variable in the very next line had the same name. This caused confusion about which `_callbackUrl` the variable `uri` was referring to. Also, Dart suggested avoiding parameters with underscores.

We can change the variable name from `callbackURL` to something else